d3a_api_client/redis_aggregator.py

```diff
@@ -182,49 +189,55 @@ def execute_batch_commands(self, is_blocking=True):
                 raise RedisAPIException(f'API registration process timed out.')
 
     def _on_event_or_response(self, message):
-        logging.info(f"A new message was received. Message information: {message}")
+        log_msg = copy(message)
+        if 'grid_tree' in log_msg:
+            log_msg['grid_tree'] = '{...}'
+        logging.debug(f"A new message was received. Message information: {log_msg}")
         log_market_progression(message)
-        function = lambda: self.on_event_or_response(message)
-        self.executor.submit(execute_function_util, function=function,
+        self.executor.submit(execute_function_util,
+                             function=lambda: self.on_event_or_response(message),
                              function_name="on_event_or_response")
 
     def calculate_grid_fee(self, start_market_or_device_name: str,
                            target_market_or_device_name: str = None,
-                           fee_type: str = "next_market_fee"):
+                           fee_type: str = "current_market_fee"):
         return self.grid_fee_calculation.calculate_grid_fee(start_market_or_device_name,
                                                             target_market_or_device_name, fee_type)
 
+    def _create_area_name_uuid_mapping(self):
+        self.area_name_uuid_mapping = \
+            create_area_name_uuid_mapping_from_tree_info(self.latest_grid_tree_flat)
```

Comment:
This method seems like an overkill as  `create_area_name_uuid_mapping_from_tree_info ` could directly be called too where `_create_area_name_uuid_mapping ` method is called.

Reply:
you are right, will change this.